homeassistant/components/netatmo/climate.py

```diff
@@ -170,7 +171,9 @@ def __init__(self, netatmo_device: NetatmoRoom) -> None:
             ]
         )
 
-        self._model: str = f"{self._room.climate_type}"
+        if not self._room.climate_type:
+            raise ValueError("No climate type found for this room")
```

Comment:
Do we want to raise an error? Or should we move this check to the place where we create entities and avoid creating entities that does not have a climate type?

Reply:
That would make sense indeed @joostlek 👍